src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.controller;
+
+import javax.validation.Valid;
+import mate.academy.model.User;
+import mate.academy.model.dto.UserRegistrationDto;
+import mate.academy.model.dto.UserResponseDto;
+import mate.academy.security.AuthenticationService;
+import mate.academy.service.mapper.UserMapper;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserMapper userMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
```

Comment:
```suggestion
    @Autowired
    public AuthenticationController(AuthenticationService authenticationService,
```
for consistency, we should better use annotation everywhere

Reply:
not sure this is required. I haven't used @Autowired over constructors